Rakefile

```diff
@@ -24,6 +24,16 @@ namespace :test do
     sh "xcodebuild test -configuration Release -scheme #{SCHEME} -destination '#{DESTINATION}' | xcpretty -c; exit ${PIPESTATUS[0]}"
   end
 
+  desc "Runs the unit tests for Scenarios via swiftpm"
+  task :package do
+    begin
+      cp ".Package.test.swift", "Package.swift"
+      sh "swift test"
+    ensure
+      sh "git checkout HEAD -- Package.swift"
```

Comment:
Most of the time it's equivalent, but there are times where git gets confused and can misinterpret a path as a branch or some other argument. The `--` is a way to unambiguously say that what follows should be treated as a path. Probably doesn't make a difference in this specific case.